### PR TITLE
Test transaction config for execute_query

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: nutkit/frontend/driver.py [hunk 1]
```diff
@@ -101,13 +101,15 @@ def send_and_receive(self, req, timeout=None, hooks=None, *,
         return self.receive(timeout=timeout, hooks=hooks,
                             allow_resolution=allow_resolution)
 
-    def execute_query(self, cypher, params=None,
-                      routing=None, database=None,
-                      impersonated_user=None, bookmark_manager=...):
+    def execute_query(self, cypher, *, params=None, routing=None,
```
Comment (bigmontz) [inline nutkit/frontend/driver.py:104]: Params should be a positional argument or tests which uses it as positional should be updated.

**Review by bigmontz — CHANGES_REQUESTED** [review]
Make `params` not positional is causing tests failure with `TypeError: execute_query() takes 2 positional arguments but 3 were given`

File: nutkit/frontend/driver.py (hunk 1 above)
Comment (robsdedude) [reply nutkit/frontend/driver.py:104]: Of course :facepalm: 

Fixed it!

**Review by bigmontz — APPROVED** [review]
🩰

**Review by thelonelyvulpes — APPROVED** [review]

**Pull request merged**